src/flavors/raingardens/config.yml

```diff
@@ -158,6 +159,36 @@ place_types:
           iconAnchor: [25, 60]
           shadowSize: [50, 60]
           shadowAnchor: [14, 60]
+  idea:
+    label: _(idea)
```

Comment:
Cool, good point! I will test and merge it soon as I finish the cluster feature (hopefully tonight/early tomorrow)

Keep in mind, we may eventually add a legend or implement the foldout legend, so we may have to revisit the capitalization issue again. At that point, I think we should consider a raingarden as a proper noun and capitalize it.